modules/api/src/main/java/org/apache/ignite/table/manager/TableManager.java

```diff
@@ -15,10 +15,42 @@
  * limitations under the License.
  */
 
+/**
+ * Interface for manage tables.
+ */
 package org.apache.ignite.table.manager;
 
+import java.util.List;
+import java.util.function.Consumer;
+import org.apache.ignite.configuration.schemas.table.TableChange;
+import org.apache.ignite.table.Table;
+
 /**
  * Interface that provides methods for managing tables.
  */
 public interface TableManager {
```

Comment:
Cause it's a part of ignite-api, should we use IgniteTables or Tables instead of exposing therm Manager externally.
I mean that we still should have TableMangerImpl -> TableManager that will implement IgniteTables.

Reply:
Renamed.